src/transformStateWithClones.js

```diff
@@ -7,7 +7,35 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const stateCopy = JSON.parse(JSON.stringify(state));
+
+  const states = [];
```

Comment:
format properly + you can use spread operator
```suggestion
  const stateCopy = { ...state };
  const states = [];
```